src/main/java/uk/co/bconline/ndelius/service/impl/RoleServiceImpl.java

```diff
@@ -90,6 +94,15 @@ public RoleServiceImpl(RoleRepository roleRepository,
 				.collect(toSet());
 	}
 
+	public Set<RoleAssociationEntry> getUsersRoles(String role)
```

Comment:
Would this be better placed in the `UserRoleService`? And could it have a more descriptive name - e.g. `getAllUsersWithRole`?